packages/meteor-accounts-saml/saml_server.js

```diff
@@ -103,8 +103,8 @@ Accounts.registerLoginHandler(function(loginRequest) {
 	}
 
 	if (loginResult && loginResult.profile && loginResult.profile.email) {
-		const email = RegExp.escape(loginResult.profile.email);
-		const emailRegex = new RegExp(`^${ email }$`, 'i');
+		const emailList = Array.isArray(loginResult.profile.email) ? loginResult.profile.email : [loginResult.profile.email];
```

Comment:
in this cases I usually do something like: `const emailList = [].concat(loginResult.profile.email);`